service/history/queues/executable.go

```diff
@@ -494,6 +494,14 @@ func (e *executableImpl) HandleErr(err error) (retErr error) {
 		}
 	}()
 
+	metricsHandler := e.taggedMetricsHandler.WithTags(metrics.ServiceErrorTypeTag(err))
```

Comment:
Make sure this metric always has the same number of tags. Prometheus complains about variable tags. Also need to check what the process is to add tags to existing metrics.